packages/tx-construction/src/tx-builder/types.ts

```diff
@@ -11,7 +11,10 @@ import { OutputBuilderValidator } from './OutputBuilder';
 import { OutputValidation } from '../output-validation';
 
 export type PartialTxOut = Partial<
-  Pick<Cardano.TxOut, 'address' | 'datumHash' | 'datum' | 'scriptReference'> & { value: Partial<Cardano.Value> }
+  Pick<Cardano.TxOut, 'address' | 'datumHash' | 'datum' | 'scriptReference'> & {
+    value: Partial<Cardano.Value>;
+    handles: Array<Handle>;
```

Comment:
A single TxOut should only refer to 1 handle, since it's to a single address
```suggestion
    handle: Handle;
```

Reply:
`handle:` or `handles:`

Reply:
[f293dc8313](https://github.com/input-output-hk/cardano-js-sdk/pull/732/commits/f293dc8313a0c07bc570e18d025412315abc45ef) should solve this